tests/server/extensions/test_chanjo_extension.py

```diff
@@ -1,44 +1,42 @@
-import requests
 from flask import Blueprint
 
-from scout.server.extensions import chanjo_report
+from scout.server.extensions import chanjo_report  # already an instance
```

Comment:
Hm, perhaps remove this comment as well, or add to the docstrings where relevant!